sql/core/src/test/scala/org/apache/spark/sql/execution/CoalesceShufflePartitionsSuite.scala

```diff
@@ -331,7 +331,8 @@ class CoalesceShufflePartitionsSuite extends SparkFunSuite with BeforeAndAfterAl
       }.length == 2)
       assert(
         finalPlan.collect {
-          case r @ CoalescedShuffleReader() => r
+          // only 1 partition - nothing to coalesce
+          case r: CustomShuffleReaderExec if !r.isLocalReader && !r.hasSkewedPartition =>
```

Comment:
is it possible to tune the test and make coalesce happen, e.g. `spark.range(0, 100, 1, numPartitions = 10)`

Reply:
done